senatus/package.json

```diff
@@ -19,6 +19,7 @@
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
     "react-router-dom": "5.3.4",
+    "sqlite3": "^5.1.7",
```

Comment:
We don't really need this as a dependency, we have a docker setup ready to go for local development that mirrors production.